source/acquaman/AMScanActionControllerBasicFileWriter.cpp

```diff
@@ -62,6 +65,12 @@ AMScanActionControllerBasicFileWriter::AMScanActionControllerBasicFileWriter(con
 		if(!spectraFile_->open(QIODevice::WriteOnly | QIODevice::Text))
 			emitError(AMScanActionControllerBasicFileWriter::CouldNotOpenError);
 	}
+
+	// start the timer (2mins) to check whether we are writing to the files consistently
+	QTimer *dataFileSizeCheckTimer = new QTimer(this);
```

Comment:
Yes, you are correct, @XuetaoLiu-CLS.  That was my mistake.  I didn't see the ```this``` pointer.